### ISSUE TITLE
onboarding map is not a function

### ISSUE THREAD

**Issue by jahwood** [avocode/avocode#2007]
---
* **Version**: 2.24.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 281688
---
```
TypeError: (r || []).map is not a function
    at file:///C:/Users/ivan/AppData/Local/avocode/app-2.24.0/resources/app.asar/dist/avocode.min.js:1:3559274
    at file:///C:/Users/ivan/AppData/Local/avocode/app-2.24.0/resources/app.asar/dist/avocode.min.js:1:2421100
    at <anonymous>
```

**Comment by jakubzitny** [comment]
Hi, thanks for reporting this. It's a bug and we will fix it for the next release.

**Comment by jakubzitny** [comment]
Hi @jahwood, this bug is fixed in latest Avocode 2.25.0. Please update and let us know if you have any other issues 😉
